test/rules/parameter_assignments_test.dart

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) 2022, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:test_reflective_loader/test_reflective_loader.dart';
+
+import '../rule_test_support.dart';
+
+main() {
+  defineReflectiveSuite(() {
+    defineReflectiveTests(ParameterAssignmentsTest);
+  });
+}
+
+@reflectiveTest
+class ParameterAssignmentsTest extends LintRuleTest {
```

Comment:
There should be one test for assignment of parameters of local functions.

Reply:
Added.  Thanks!